codec/types/any.go

```diff
@@ -139,8 +125,3 @@ func (any *Any) GetCachedValue() interface{} {
 func (any *Any) ClearCachedValue() {
 	any.cachedValue = nil
 }
-
-// IntoAny represents a type that can be wrapped into an Any.
-type IntoAny interface {
```

Comment:
As discussed yesterday, maybe we could rename it, e.g. `TxBuilderToAny`, and move it somewhere less top-level.

Reply:
?, this interface is moved and it's private.

Reply:
I wrote this comment before our conversation last evening.